content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,88 @@
 ---
 title: Getting Started
-# prettier-ignore
-cSpell:ignore: darwin dpkg GOARCH journalctl kubectl otelcorecol pprof tlsv zpages
+cSpell:ignore: docker dokey dpkg okey telemetrygen
 weight: 1
 ---
 
-If you aren't familiar with the deployment models, components, and repositories
-applicable to the OpenTelemetry Collector, first review the [Data Collection][]
-and [Deployment Methods][] page.
+The OpenTelemetry Collector listens to [traces](/docs/concepts/signals/traces/),
+[metrics](/docs/concepts/signals/metrics/) and
+[logs](/docs/concepts/signals/logs/), processes the telemetry, and exports it to
+a wide variety of observability back-ends using its components. For a conceptual
```

Comment:
```suggestion
a wide variety of observability backends using its components. For a conceptual
```

One more to discuss, we use `backend` more often than `back-end` throughout the docs

Reply:
Right! FWIW, here's what the Splunk style guide has to say on the matter: https://docs.splunk.com/Documentation/StyleGuide/current/StyleGuide/Usagedictionary#B

OTOH, Google goes the other way: https://developers.google.com/style/word-list#letter-b

Reply:
Yeah, there is no consistency. I looked up k8s website, they have a preference for `backend` but some instances of `back end` and `back-end` as well, so we don't need to be religious here